events/omero-workshop-eastbio-dundee-november-2021.html

```diff
@@ -0,0 +1,42 @@
+---
+layout: news-events
+filename: events
+title: November 2021 OMERO EastBio workshop
+main-blurb: 1-hour online demo/workshop on how to use OMERO.
+---          
+     <hr class="whitespace">
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">The OMERO demo/workshop is a part of a whole-day course for Postdoctoral Students.</h2>
+            </div>
+        </div>
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Organiser: <a href="http://www.eastscotbiodtp.ac.uk/" target="_blank">EastBio</a></h2>
+            </div>
+        </div>
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Location: Dundee</h2>
```

Comment:
```suggestion
                <h2 class="event-day">Location: online</h2>
```

suggesting "online" as this was the formulation on https://www.openmicroscopy.org/events/omero-users-workshop-eastbio-edinburgh-online-november-2020.html. More precisely, it is held from Edinburgh Uni remotely, but "online" should do imho